libr/core/cmd_meta.c

```diff
@@ -68,6 +68,7 @@ static const char *help_msg_Cs[] = {
 	"Cs", "", "list all strings in human friendly form",
 	"Cs*", "", "list all strings in r2 commands",
 	"Cs", " [size] @addr", "add string (guess latin1/utf16le)",
+	" Csg", " [size] [@addr]", "as above but addr not needed",
```

Comment:
why this space after "

Reply:
personal taste ... the space isn't needed and I have removed it